### PR TITLE
Update gulp-nsp

### PR DESCRIPTION
**Pull request by mieszko4**
.Before this fix I was getting the following error when running `gulp publish`:
```
return m ? m.index + 1 : c.length;
                              ^
TypeError: Cannot read property 'length' of undefined
```

This pull request fixes that error. I also believe it should fix the follwing issue: https://github.com/yeoman/generator-node/issues/180

### CODE REVIEW

File: generators/gulp/templates/gulpfile.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ gulp.task('static', function () {
 });
 
 gulp.task('nsp', function (cb) {
-  nsp('package.json', cb);
+  gulpNSP({package: __dirname + '/package.json'}, cb);
```
Comment (SBoudrias) [inline generators/gulp/templates/gulpfile.js]: Can we return rather than using a callback?

File: generators/gulp/templates/gulpfile.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ var eslint = require('gulp-eslint');
 var excludeGitignore = require('gulp-exclude-gitignore');
 var mocha = require('gulp-mocha');
 var istanbul = require('gulp-istanbul');
-var nsp = require('gulp-nsp');
+var gulpNSP = require('gulp-nsp');
```
Comment (SBoudrias) [inline generators/gulp/templates/gulpfile.js]: Why renaming?

**Comment by mieszko4** [comment]
I've basically used usage example from https://github.com/nodesecurity/gulp-nsp
It seems they have new API now.

**Comment by SBoudrias** [comment]
Okay, just keep the previous `nsp` name.

**Comment by hemanth** [comment]
Thanks.

**Comment by hemanth** [comment]
@SBoudrias Could you please do a release? I am AFK...

**Comment by voltrevo** [comment]
@hemanth Yep works for me now :+1:.

**Pull request merged**